HomeWorks/HW4/Routers/Network.cs

```diff
@@ -0,0 +1,81 @@
+namespace Routers;
+
+public class Network(List<Edge> edges)
+{
+    private List<Edge> edges = edges;
+    private HashSet<int> nodes = new HashSet<int>(edges.SelectMany(e => new[] { e.From, e.To }));
```

Comment:
```suggestion
    private HashSet<int> nodes = new(edges.SelectMany(e => new[] { e.From, e.To }));
```
Не пишите один тип дважды